mainnet-1/specs/stargaze.json

```diff
@@ -0,0 +1,787 @@
+{
+  "proposal": {
+    "title": "Add Specs: Stargaze",
+    "description": "Adding new specification support for relaying agoric data on stargaze",
+    "specs": [
+      {
+        "index": "STRGZ",
+        "name": "stargaze mainnet",
+        "enabled": true,
+        "imports": [
+          "COSMOSSDK50",
+          "COSMOSWASM"
+        ],
+        "reliability_threshold": 268435455,
+        "data_reliability_enabled": true,
+        "block_distance_for_finalized_data": 0,
+        "blocks_in_finalization_proof": 1,
+        "average_block_time": 5800,
+        "allowed_block_lag_for_qos_sync": 2,
+        "shares": 1,
+        "min_stake_provider": {
+          "denom": "ulava",
+          "amount": "15000000000"
```

Comment:
This should be 5K lava (= 5000000000 ulava)

Reply:
The field value has been updated [f945313](https://github.com/lavanet/lava-config/pull/24/commits/f945313bd76273364a1dd62f8aa8357432ed92d8)